code/index.html

```diff
@@ -3,39 +3,66 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <meta name="description" content="This is a boilerplate for building websites"/>
+    <meta name="beached whale" content="beached whale adventure apparel and knickknacks"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Trad Is Rad</title>
-    <link href='http://fonts.googleapis.com/css?family=Amatic+SC' rel='stylesheet' type='text/css'>
+
+  <!--- FONTS -------------------->
+    <link href="http://fonts.googleapis.com/css?family=Amatic+SC:400,700" rel="stylesheet" type="text/css">
+    <link href="http://fonts.googleapis.com/css?family=Open+Sans+Condensed:300,700" rel="stylesheet" type="text/css">
+  <!--- stylesheets--------------------->
     <link rel="stylesheet" href="css/reset.css" type="text.css" />
+
     <link rel="stylesheet" href="css/style.css" type="text/css"/>
+
+    <link rel="stylesheet" media="only screen and (max-width: 600px)" href="css/mobile.css" type="text/css"/>
+
     <!-- adding google fonts <script src="url"></script> -->
   </head>
 
   <body>
     <header>
-      <div class="container">
+      <div class="headercontainer">
```

Comment:
When writing css use snake casing between two words "-". header-container. Accepted styling practice and easier to read.